packages/protocol/socket.ts

```diff
@@ -217,11 +217,6 @@ export async function sendMessage<M extends CommandMethods>(
 
     const { code, data, message } = response.error;
 
-    if (method === "Session.listenForLoadChanges" && code === 66) {
-      // We are being disconnected after a timeout, no need to raise
-      return {};
-    }
```

Comment:
Now that we're listening for these exceptions explicitly, we shouldn't need to special case them here any more.